### PR TITLE
Adding BCD for fullscreen events

### PR DESCRIPTION
**Pull request by irenesmith**
This is for [MDN/Sprints](https://app.zenhub.com/workspaces/mdn-sprint-board-5ab3d23fd0f4ea53b0022a61/issues/mdn/sprints/1104) - Event refactoring of fullscreenchange and fullscreenerror events on the Document and Element objects. Data is based on the data for corresponding event handler properties.

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -4210,6 +4210,92 @@
           }
         }
       },
+      "fullscreenchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenchange_event",
+          "description": "<code>fullscreenchanged</code> event"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
          "description": "<code>fullscreenchanged</code> event",
```

File: api/Element.json [hunk 2]
```diff
@@ -1199,6 +1199,132 @@
...
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "fullscreenerror_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/fullscreenerror_event",
+          "description": "<code>fullsecreenerror</code> event"
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
          "description": "<code>fullsecreenerror</code> event",
```

File: api/Document.json [hunk 3]
```diff
@@ -4210,6 +4210,92 @@
...
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "47",
+                "version_removed": "65",
```
Comment (vinyldarkscratch) [inline api/Document.json]: I'm thinking that these `version_removed: 65` statements should actually be 64.  Based upon your data, Firefox 64 was the version where it was no longer hidden behind a flag; thus, the requirement to use the flag was removed in Firefox 64.

File: api/Document.json [hunk 4]
```diff
@@ -4210,6 +4210,92 @@
...
+                "alternative_name": "onmozfullscreenchange"
+              }
+            ],
+            "ie": {
+              "version_added": "11",
+              "alternative_name": "onMSFullscreenChange"
+            },
+            "opera": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "32"
```

File: api/Document.json [hunk 5]
```diff
@@ -4210,6 +4210,92 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "32"
```

File: api/Document.json [hunk 6]
```diff
@@ -4313,6 +4399,92 @@
...
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenerror"
+              }
+            ],
+            "ie": {
+              "version_added": "11",
+              "alternative_name": "onMSFullscreenError"
+            },
+            "opera": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "32"
```

File: api/Document.json [hunk 7]
```diff
@@ -4313,6 +4399,92 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "32"
```

File: api/Element.json [hunk 8]
```diff
@@ -1199,6 +1199,132 @@
...
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenchange"
+              }
+            ],
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "44"
```

File: api/Element.json [hunk 9]
```diff
@@ -1199,6 +1199,132 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "43"
```

File: api/Element.json [hunk 10]
```diff
@@ -1199,6 +1199,132 @@
...
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenerror"
+              }
+            ],
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "44"
```

File: api/Element.json [hunk 11]
```diff
@@ -1199,6 +1199,132 @@
...
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "43"
```

File: api/Document.json (hunk 1 above)
Comment (wbamberg) [reply api/Document.json]: Also I think this should be `<code>fullscreenchange</code>`, not `<code>fullscreenchanged</code>`.

File: api/Document.json [hunk 12]
```diff
@@ -4210,6 +4210,92 @@
...
+                    "type": "preference",
+                    "name": "full-screen-api.unprefix.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenchange"
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenchange#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenchange` form.

File: api/Document.json [hunk 13]
```diff
@@ -4210,6 +4210,92 @@
...
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenchange#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenchange` form.

File: api/Document.json [hunk 14]
```diff
@@ -4210,6 +4210,92 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": "11",
+              "alternative_name": "onMSFullscreenChange"
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenchange#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenchange` form.

File: api/Document.json [hunk 15]
```diff
@@ -4313,6 +4399,92 @@
...
+                    "type": "preference",
+                    "name": "full-screen-api.unprefix.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenerror"
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenerror#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenerror` form.

File: api/Document.json [hunk 16]
```diff
@@ -4313,6 +4399,92 @@
...
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenerror#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenerror` form.

File: api/Document.json [hunk 17]
```diff
@@ -4313,6 +4399,92 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": "11",
+              "alternative_name": "onMSFullscreenError"
```
Comment (wbamberg) [inline api/Document.json]: This data is copied from https://developer.mozilla.org/en-US/docs/Web/API/Document/onfullscreenerror#Browser_compatibility so the alternative name has the `on-` prefix, but that's not correct for the `fullscreenerror` form.

File: api/Element.json [hunk 18]
```diff
@@ -1199,6 +1199,132 @@
...
+            },
+            "firefox": [
+              {
+                "version_added": "64"
+              },
+              {
+                "version_added": "10",
+                "version_removed": "65",
+                "alternative_name": "onmozfullscreenchange"
```
Comment (wbamberg) [inline api/Element.json]: All the comments about this `on-` alternative name in the Document.json file also apply to the Element.json file.

**Comment by irenesmith** [comment]
I've worked through both files and updated according to suggestions. Thanks!

File: api/Document.json [hunk 19]
```diff
@@ -4210,6 +4210,92 @@
           }
         }
       },
+      "fullscreenchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/fullscreenchange_event",
+          "description": "<code>fullscreenchanged</code> event",
```
Comment (wbamberg) [inline api/Document.json]: This still needs to be "fullscreenchange".

File: api/Element.json [hunk 20]
```diff
@@ -1199,6 +1199,132 @@
...
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "fullscreenerror_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/fullscreenerror_event",
+          "description": "<code>fullsecreenerror</code> event",
```
Comment (wbamberg) [inline api/Element.json]: Typo "fullsecreenerror" -> "fullscreenerror"

File: api/Element.json [hunk 21]
```diff
@@ -2900,7 +3026,7 @@
               {
                 "version_added": "10",
                 "version_removed": "65",
-                "alternative_name": "onmozfullscreenchange"
+                "alternative_name": "mozfullscreenchange"
```
Comment (wbamberg) [inline api/Element.json]: Is this change correct? This subfeature is under "onfullscreenchange" not "fullscreenchange_event"

File: api/Document.json [hunk 22]
```diff
@@ -5959,7 +6131,7 @@
               {
                 "version_added": "10",
                 "version_removed": "65",
-                "alternative_name": "onmozfullscreenchange"
+                "alternative_name": "mozfullscreenchange"
```
Comment (wbamberg) [inline api/Document.json]: It looks this PR removes the `on-` prefix from alternative names even for the `on-` event handler property, which I don't think is correct.

**Comment by vinyldarkscratch** [comment]
Closing this in favor of #4054.

**Pull request closed without merging**